manager.go

```diff
@@ -113,7 +109,6 @@ func newManager(queue string, job JobFunc, concurrency int, mids ...Action) *man
 		make(chan *Msg),
 		make(chan bool),
 		make(chan bool),
-		customMids,
```

Comment:
Somewhat related to my other comment about `job`. What you think about preserving the job alone + having another field for job + middleware? If think that is not necessary then I would consider renaming `job` to be more expressive of its purpose.

Reply:
I'd rather avoid having both the `job` and the `middleware + job` on the struct, since there's no real use case. I'm happy to change the name from `job` to... `handler`, perhaps? Any opinions on the new name?

Reply:
`handler` is a good name. And I concur that if there is no use case right now then we should not keep it on the struct; and if we need it later we can add it since we vacate job.